### ISSUE TITLE
eth_subscribe with websockets/ipc eventually stops sending messages

### ISSUE THREAD

**Issue by kevinday** [ethereum/go-ethereum#2887]
#### System information

```
Version: 1.5.0-unstable-c8a8ad97
Protocol Versions: [63 62 61]
Network Id: 1
Go Version: go1.6.2
OS: darwin
```

I'm attempting to use `eth_subscribe` + `logs` with websockets and/or ipc, and I'm noticing it happily sends the events along for a while, but eventually stops receiving subscription messages (usually after a few minutes). The websocket connection seems to still be open as I still receive responses for other requests.

My subscription request looks like this:

```
{
    "id": 1,
    "jsonrpc": "2.0",
    "method": "eth_subscribe",
    "params": ["logs", {
        "address": "0xea3fc35beafe2dff9efa562b903f4695c26a60dc",
        "topics": ["0x63f140d7adcc464732c9379020aa9e5ce1b1e350796814d780ea3ca41d62a36b"]
    }]
}
```

I establish a websocket connection, and then have a script that triggers the log on the contract (https://github.com/AugurProject/augur-core/blob/master/src/functions/createMarket.se#L174) every 30 seconds or so. If I kill my websocket connection and resend the `eth_subscribe`, the subscription messages (temporarily) come back.

**Comment by bas-vk** [comment]
Can confirm this is an issue and is fixed in https://github.com/ethereum/go-ethereum/pull/2885.

**Comment by kevinday** [comment]
Excellent news, thanks bas

**Comment by fjl** [comment]
Fix released in 1.5.0.
